app/frontend/src/components/Answer/AnswerParser.tsx

```diff
@@ -20,6 +20,21 @@ export function parseAnswerToHtml(answer: string, onCitationClicked: (citationFi
     // trim any whitespace from the end of the answer after removing follow-up questions
     parsedAnswer = parsedAnswer.trim();
 
+    // Omit a citation that is still being typed during streaming
+    if (isStreaming){
```

Comment:
I assume this is the part that changed after I last looked at it, this looks reasonable, thanks for addressing that!